common/app/views/fragments/email/signup/emailIframe.scala.html

```diff
@@ -9,4 +9,4 @@
     id="@{formType + "__email-form"}" frameborder="0"
     class="iframed--overflow-hidden email-sub__iframe js-email-sub__iframe"
     data-form-success-headline="We've sent you an email!"
-    data-form-success-desc="Open up the email and click the verify button to confirm your email address.<br/><br/>You'll then start receiving the daily email."></iframe>
+    data-form-success-desc="Open up the email and click the confirm button to verify your email address.<br/><br/>You'll then start receiving the daily email."></iframe>
```

Comment:
I would say it's more that we're validating their subscription attempt than verifying their email address, but I'm not sure a user will understand or care about this distinction anyway.